examples/pipeline/1_basic_example.py

```diff
@@ -29,15 +30,17 @@
 """
 
 
+# %%
 pipeline = Pipeline.from_script(
     SCRIPT,  # Actor script object, defined in `.utils` module.
     start_label=("greeting_flow", "start_node"),
     fallback_label=("greeting_flow", "fallback_node"),
 )
 
 
+# %%
 if __name__ == "__main__":
-    if get_auto_arg():
+    if should_auto_execute():
```

Comment:
pls, will rename in future PR

Reply:
It will be completely reworked in future PRs